x-pack/plugins/security_solution/public/transforms/constants.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/**
+ * TODO: Once this is moved into the plugin then this should use the
+ * ELASTIC_NAME from the plugin
+ */
+export const ELASTIC_NAME = 'estc';
```

Comment:
Move to `common`?

Reply:
Yeah, I can move this to common constants.